code/game/objects/items/weapons/grenades/chem_grenade.dm

```diff
@@ -184,8 +184,8 @@
 	if (E != null)
 		extract_uses = E.Uses
 		for(var/i=1,i<=extract_uses,i++)//<-------//exception for slime extracts injected with steroids. The grenade will repeat its checks untill all its remaining uses are gone
-			if (reservoir.reagents.has_reagent("plasma", 5))
-				reservoir.reagents.trans_id_to(E, "plasma", 5)		//If the grenade contains a slime extract, the grenade will check in this order
+			if (reservoir.reagents.has_reagent(PLASMA, 5))

```

Comment:
inappropriate replacement (reagent ID, those need to be refactored separately)
same for the rest of this file